docs/README.md

```diff
@@ -38,9 +38,9 @@ order: 6
 ```
 
 
-### Code Examples
```

Comment:
we should explain what a "snippet" is too (it's not as self-descriptive as the previous name)